<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseFeaturesWindow</class>
 <widget class="QDialog" name="ChooseFeaturesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="areaCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Area</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="perimetrCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Perimetr</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="compactnessCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Compactness</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="lumaCheckBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Luma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
